### DISCUSSION TITLE
Vercel deployment GitHub settings not working as expected

### DISCUSSION THREAD

**Discussion by jakeleventhal** [vercel/community#2328] [Bug Report]
I have a repo with 3 applications. The root of my repo has a `vercel.json` with the following contents:
```json
{
  "github": {
    "deploymentEnabled": false,
    "silent": true
  }
}
```

I don't get comments in my PRs due to the `silent` field, but it seems like `deploymentEnabled` is not working as expected. For each application that I have, I have the ignore build step setting set to `true` so that it always skips (since I handle this work in CI).
<img width="1046" alt="Screenshot 2023-05-05 at 8 04 55 PM" src="https://user-images.githubusercontent.com/15133828/236587274-6c873670-6ff1-4d69-a5ff-db745e4957ce.png">

I would expect that when I make a PR, those checks never show up at all. Is there a way to hide those so that I don't have a bunch of dummy checks that always say ignored? Also, is there a way to disable the check on preview comments?

**Comment by GRA0007** [comment]
We also use Github Actions to build preview deployments so that we can run Cypress tests on them afterwards (as there's no way to reliably wait for a Vercel build to finish in our tests), and it's confusing that Vercel always marks each build as cancelled by ignored build step next to where our test results are displayed.
